VEHICLE/IsVehicleSirenOn.md

```diff
@@ -7,9 +7,18 @@ ns: VEHICLE
 // 0x4C9BF537BE2634B2 0x25EB5873
 BOOL IS_VEHICLE_SIREN_ON(Vehicle vehicle);
 ```
-
+Returns whether the vehicle's lights and sirens are on.
 
 ## Parameters
 * **vehicle**: 
 
 ## Return value
+Returns false if siren and lights are disabled or 1 if they enabled.
```

Comment:
Typo 😭 
"if they enabled" > "if they're enabled"

Reply:
the return value shouldnt be 1 anyway, as mentioned its purely for compatability, its just a truthy value and used correctly the value doesnt matter, its just true/false